rpcs3/Emu/system_config.h

```diff
@@ -294,9 +294,10 @@ struct cfg_root : cfg::node
 		cfg::_enum<CellKbMappingType> keyboard_type{ this, "Keyboard Type", CellKbMappingType{0} }; // CELL_KB_MAPPING_101 = US
 		cfg::_enum<enter_button_assign> enter_button_assignment{ this, "Enter button assignment", enter_button_assign::cross };
 		cfg::_int<-60*60*24*365*100LL, 60*60*24*365*100LL> console_time_offset{ this, "Console time offset (s)", 0 }; // console time offset, limited to +/-100years
-		cfg::uint<0,umax> console_psid_high{ this, "PSID high"};
-		cfg::uint<0,umax> console_psid_low{ this, "PSID low"};
-
+		cfg::uint<0, umax> console_psid_high{this, "PSID high"};
+		cfg::uint<0, umax> console_psid_low{this, "PSID low"};
+		cfg::uint<0, umax> lv2_board_storage_high{this, "LV2 Board Storage high"};
+		cfg::uint<0, umax> lv2_board_storage_low{this, "LV2 Board Storage low"};
```

Comment:
why do they need to be settings? Is it something that should persist?

Reply:
Yes, it should persists and we should let users decide which value to put.